packages/docs/liveEditorCode/alert.code.js

```diff
@@ -3,13 +3,11 @@
   return dismissed ? null : (
     <Alert
       dismissible
-      size={Alert.Size.SMALL}
-      arrow={Alert.ArrowPosition.TOP_LEFT}
-      type={Alert.Type.WARNING}
+      size={Alert.Size.LARGE}
```

Comment:
MAybe we don't want to encourage LARGE
```suggestion
      size={Alert.Size.SMALL}
```